collect_app/src/main/java/org/odk/collect/android/fragments/ShowQRCodeFragment.java

```diff
@@ -118,15 +117,11 @@ private void addPasswordStatusString() {
     }
 
     private void updateShareIntent(Bitmap qrCode) throws IOException {
-
-        //Save the bitmap to a file
-        File shareFile = saveBitmapToCache(qrCode);
-
-        // Sent a intent to share saved image
+        // Send a intent to share saved image
         shareIntent = new Intent();
         shareIntent.setAction(Intent.ACTION_SEND);
         shareIntent.setType("image/*");
-        shareIntent.putExtra(Intent.EXTRA_STREAM, Uri.parse("file://" + shareFile));
+        shareIntent.putExtra(Intent.EXTRA_STREAM, Uri.parse("file://" + saveBitmapToCache(qrCode)));
```

Comment:
I think this makes the code harder to read. What about just moving the variable to the line before?